registration/models.py

```diff
@@ -142,6 +142,7 @@ def resend_activation_mail(self, email, site, request=None):
             profile = self.get(user__email=email)
         except ObjectDoesNotExist:
             return False
+        # TODO: Catch multiple objects returned exception?
```

Comment:
That makes sense! Thanks for the clarification.

You are right. Emails are not unique by default in Django and django-registration. This is beyond ready to merge :smile: so let's get this in into master and I will file an issue to discuss the email uniqueness TODO and we can discuss.